src/Compression/CompressedWriteBuffer.h

```diff
@@ -13,7 +13,7 @@
 namespace DB
 {
 
-class CompressedWriteBuffer final : public BufferWithOwnMemory<WriteBuffer>
+class CompressedWriteBuffer : public BufferWithOwnMemory<WriteBuffer>
```

Comment:
I did not understand why should it be final. Do you understand it?

At some point I make some instance as `AutoCanceledWriteBuffer< CompressedWriteBuffer >` and it did not work because `CompressedWriteBuffer` was marked as `final. So I removed it.

Reply:
Makes sense to me